terrawrap/utils/config.py

```diff
@@ -302,6 +295,24 @@ def calc_backend_config(
     return backend_config
 
 
+def calc_repo_path(path: str) -> str:
```

Comment:
Maybe move this to https://github.com/amplify-education/terrawrap/blob/master/terrawrap/utils/path.py